app/javascript/application.js

```diff
@@ -1,3 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+// import "custom/main"
```

Comment:
Code that you do not use, do not comment on it, delete it.

Reply:
Perfect, we will eliminate the unnecessary code.